packages/admin/cms-admin/src/dependencies/Dependency.tsx

```diff
@@ -0,0 +1,106 @@
+import { useQuery } from "@apollo/client";
+import { OperationVariables } from "@apollo/client/core";
+import { Link as LinkIcon, OpenNewTab as OpenNewTabIcon } from "@comet/admin-icons";
+import { CircularProgress, IconButton, ListItem, ListItemProps, ListItemSecondaryAction, ListItemText } from "@mui/material";
+import makeStyles from "@mui/styles/makeStyles";
+import * as React from "react";
+import { FormattedMessage } from "react-intl";
+import { Link } from "react-router-dom";
+
+import { DependencyInterface } from "../documents/types";
+import { GQLDependency } from "../graphql.generated";
+
+export interface DependencyComponentProps {
+    id: string;
+    dependencyData: Pick<GQLDependency, "rootColumnName" | "jsonPath">;
+    contentScopeUrl: string;
+}
+export type DependencyComponentInterface = (props: DependencyComponentProps) => React.ReactElement<ListItemProps>;
+
+const useStyles = makeStyles((theme) => ({
+    root: {
+        minHeight: 61,
+    },
+    displayNameColumn: {
+        flexGrow: 0,
+        paddingRight: theme.spacing(4),
+    },
+}));
```

Comment:
AFAIK the useStyles/makeStyles API is deprecated, please use styled or even the sx prop